Makefile

```diff
@@ -145,7 +145,7 @@ autoreview: cs-check phpstan psalm validate test-autoreview rector-check detect-
 
 .PHONY: test
 test:		 	## Runs all the tests
-test: autoreview test-unit test-e2e test-infection
+test: autoreview test-unit test-benchmark test-e2e test-infection
```

Comment:
maybe we could add `test-benchmark-docker`  target too? So it will be easy to run benchmarks on different PHP versions inside of docker containers?

Reply:
out of curiosity: do you actually use `***-docker` targets?

I find myself jumping into a container e.g. `docker compose run ...` and just run usual targets there like `make test` etc., but probably that's only me

Reply:
>maybe we could add test-benchmark-docker target too?

I've just tried, but that requires more work as the coverage path would no longer match. I'm not too keen on working on that.